libs/openFrameworks/utils/ofUtils.h

```diff
@@ -735,11 +735,11 @@ char ofToChar(const string& charString);
 template <class T>
 string ofToBinary(const T& value) {
 	ostringstream out;
-	const char* data = (const char*) &value;
+	const unsigned long long* data = static_cast<unsigned long long*>(&value);
```

Comment:
is this change well tested? don't wont to merge something that might break at this point but if this fixes warnings i'll do it. also can you change unsigned long long with uint64_t?